### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Zibi95**
[DEMO LINK](https://Zibi95.github.io/layout_snake/)
[TEST REPORT LINK](https://Zibi95.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: calc(var(--block-height * 3));
+}
+
+.block {
+  height: var(--block-height);
+  flex-basis: var(--block-height);
+  line-height: var(--block-height);
```
Comment (DorotaLeniecDev) [inline src/style.css:22]: To center block content vertically and horizontally you could use flex here as well, instead line 22 and 23 you could write 
`
    display: flex;
    align-items: center;
    justify-content: center;
`

**Comment by DorotaLeniecDev** [comment]
I can see that one of the point is not finalized 

- Blocks must be put in 2 to 6 rows and stretched: each row must fit either 1, 2, or 3 blocks. See the screenshots

Please double check this :)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  --block-height: 300px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: calc(var(--block-height * 3));
```
Comment (DorotaLeniecDev) [inline src/style.css:16]: This is not a valid CSS property.  I think you wanted to set min-width so be 300px right?

**Review by DorotaLeniecDev — CHANGES_REQUESTED** [review]
Small adjustmens needed.

File: src/style.css (hunk 1 above)
Comment (Zibi95) [reply src/style.css:23]: Yea, but for one liners content this makes one property less. Should i change that?

**Review by DorotaLeniecDev — CHANGES_REQUESTED** [review]
One last change and we are good to go :)

**Review by DorotaLeniecDev — APPROVED** [review]
Good job :)

**Pull request closed without merging**